recipe/build.sh

```diff
@@ -1,27 +1,24 @@
 #!/bin/bash
 
+set -eo pipefail
+
+export PKG_CONFIG_PATH="${BUILD_PREFIX}/bin/pkg-config:${BUILD_PREFIX}/lib/pkgconfig:${PREFIX}/bin/pkg-config:${PREFIX}/lib/pkg-config"
+export PKG_CONFIG_EXECUTABLE=${PREFIX}/bin/pkg-config
+
 declare -a CMAKE_PLATFORM_FLAGS
-if [[ ${HOST} =~ .*linux.* ]]; then
-    CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
-fi
 
 mkdir build_shared && cd $_
-cmake \
+cmake -G "Ninja" \
+    ${CMAKE_ARGS} \
+    -DALLOW_IN_SOURCE_BUILD=ON \
     -DCMAKE_PREFIX_PATH=$PREFIX \
     -DCMAKE_INSTALL_PREFIX=$PREFIX \
-    -DJAS_ENABLE_AUTOMATIC_DEPENDENCIES=False \
-    ${CMAKE_PLATFORM_FLAGS[@]} ..
-make
-make install
-cd ..
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON \
```

Comment:
This isn't being used
```
None INFO   Manually-specified variables were not used by the project:
None INFO
None INFO     CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP
None INFO
None INFO
```

Reply:
Since @cbouss suggested it I wonder if something isn't happening that should be

Reply:
Removed